modules/ossm-rn-deprecated-features.adoc

```diff
@@ -15,6 +15,25 @@ Deprecated functionality is still included in {product-title} and continues to b
 
 Removed functionality no longer exists in the product.
 
+== Deprecated and removed features {SMProductName} 2.3.1
+
+The `concurrency` field of the `spec.proxy` API is deprecated as of release 2.3.1, and is planned for removal in a future release. Although the field is still supported in release 2.3.1, customers should begin using the `concurrency` field of the `ProxyConfig` API.
```

Comment:
Could we add this item to "known issues" and:
1. Explain how to workaround it with `ProxyConfig`.
2. Say that we are going to deprecate this config option in the future and recommend using `ProxyConfig`?

Reply:
Done